content/english/python-earsketch/8-activity-4-add-effects-to-your-songs/_index.md

```diff
@@ -32,59 +29,30 @@ of the two `setEffect` functions:
-    effects. By default, you should see section 30.4 BANDPASS on the
-    right. For now, we will use the DELAY from section 30.4 as
-    `effectType` in our `setEffect` function call.
-6.  Now that we have chosen our `effectType`, we need an
-    `effectParameter`. Navigate to section 30.4 on the right to check
-    out different parameters supported for DELAY. Here is a picture of
-    that section. We recommend using the DELAY\_FEEDBACK.
-    ![](../img/screenshot-effect-parameter.png)
-7.  For the final parameter, input a valid value. Since we have chosen
-    DELAY\_FEEDBACK, we must specify a number between -120.0 and -1.0
-    (as shown in the picture above). Example: Use -6.0 as your effect
-    value for DELAY\_FEEDBACK.
+3.  We need to replace the parameters with our own values. Put an integer value for `trackNumber`. Let's put 2.
+4.  For the rest of the effect parameters, click on the eye icon on the `setEffect` API and you will see a link to "Every Effect Explained in Detail". Click on it to open this section on the right side of your window.
+5.  You may notice that Earsketch has many different variations of effects. By default, you should see section 30.4 BANDPASS on the right. For now, we will use the DELAY from section 30.4 as `effectType` in our `setEffect` function call.
+6.  Now that we have chosen our `effectType`, we need an `effectParameter`. Navigate to section 30.4 on the right to check out different parameters supported for DELAY. Here is a picture of that section. We recommend using the DELAY\_FEEDBACK.
+    ![Section that shows supported parameters for DELAY_FEEDBACK](../img/screenshot-effect-parameter.png)
+7.  For the final parameter, input a valid value. Since we have chosen DELAY\_FEEDBACK, we must specify a number between -120.0 and -1.0 (as shown in the picture above). 
+
+    Example: Use -6.0 as your effect value for DELAY\_FEEDBACK.
+
+{{% /panel %}}
 
 Your code should now look something like this:
 
-    from earsketch import *
-    
-    init()
-    setTempo(120)
-    
-    fitMedia(HOUSE_DEEP_CRYSTALCHORD_001, 3, 1, 5)
-    fitMedia(HOUSE_DEEP_CRYSTALCHORD_002, 3, 5, 9)
-    
-    # Section A
-    def sectionA(startMeasure, endMeasure):
-        fitMedia(RD_UK_HOUSE__AIRYPAD_1, 1, startMeasure, endMeasure)
-        fitMedia(HOUSE_MAIN_BEAT_002, 4, startMeasure, endMeasure)
-        fitMedia(HOUSE_ROADS_BASS_001, 5, startMeasure, endMeasure)
-    
-    # Section B
-    def sectionB(startMeasure, endMeasure): 
-        fitMedia(Y37_ORGAN_1, 2, startMeasure, endMeasure)
-        fitMedia(HOUSE_ROADS_BASS_001, 5, startMeasure, endMeasure)
-        fitMedia(RD_UK_HOUSE__ARPLEAD_1, 6, startMeasure, endMeasure)
-    
-    sectionA(1, 9)
-    sectionB(9, 17)
-    sectionA(17, 25)
-    setEffect(2, DELAY, DELAY_FEEDBACK, -6.0)
-    setEffect(2, DELAY, DELAY_TIME, 1200.0)
-    
-    finish()
+![Example image of your code in Activity 4](img/myjam-activity4.png)
```

Comment:
Why are we using images now? I don't mind either but just curious.

Reply:
Just thought it was prettier and more readable